File: Validus.Console/DTO/PageTemplatesDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Validus.Console.DTO
{
    public class PageTemplatesDto
    {
        public int TeamId { get; set; }
        public int PageId { get; set; }
        public int PageStructureTemplateId { get; set; }
        public string PageStructureTemplateTitle { get; set; }
        public string PageStructureTemplateUrl { get; set; }
        public List<PageTemplateDto> SectionTemplates { get; set; }
    }
} 

File: Validus.Console/Scripts/app/Dtos/PageTemplatesDto.js
var ConsoleApp = window.ConsoleApp = window.ConsoleApp || {};

ConsoleApp.PageTemplatesDto = function(teamId, pageId, pageStructureTemplateId, pageStructureTemplateTitle, pageStructureTemplateUrl, sectionTemplates)
{
	this.TeamId = teamId;
	this.PageId = pageId;
	this.PageStructureTemplateId = pageStructureTemplateId;
	this.PageStructureTemplateTitle = pageStructureTemplateTitle;
	this.PageStructureTemplateUrl = pageStructureTemplateUrl;
	this.SectionTemplates = sectionTemplates;
};